controllers.js

```diff
@@ -0,0 +1,14 @@
+const {dataGetTopics} = require("./models")
+
+
+const getTopics = (req,res)=>{
+
+    return dataGetTopics(req,res).then(
+        (datatopics) => {
+            res.status(200).send(datatopics)
```

Comment:
we should be sending back an object with a key that we can then access in our tests. 

e.g. .send({topics: dataTopics})